src/lib/ares__addrinfo_localhost.c

```diff
@@ -209,13 +209,13 @@ static int ares__system_loopback_addrs(int aftype,
 }
 
 
-int ares__addrinfo_localhost(const char *name,
-                             unsigned short port,
-                             const struct ares_addrinfo_hints *hints,
-                             struct ares_addrinfo *ai)
+ares_status_t ares__addrinfo_localhost(const char *name,
+                                       unsigned short port,
+                                       const struct ares_addrinfo_hints *hints,
+                                       struct ares_addrinfo *ai)
 {
   struct ares_addrinfo_node *nodes = NULL;
-  int result;
+  ares_status_t result;
```

Comment:
I think it would increase readability if we stick to the same name for the general `ares_status_t` return code variable and we use `status` already in most places. So maybe rename here?